packages/rum-angular/src/apm.module.ts

```diff
@@ -23,12 +23,14 @@
  *
  */
 
-const { getBabelConfig, PACKAGE_TYPES } = require('../../dev-utils/build')
+import { InjectionToken, NgModule } from '@angular/core'
+import { RouterModule } from '@angular/router'
+import { apm, ApmBase } from '@elastic/apm-rum'
 
-module.exports = function(api) {
-  /**
-   * Reads BABEL_ENV to decide between CJS and ESM formats
-   */
-  const env = api.env()
-  return getBabelConfig(env, PACKAGE_TYPES.ANGULAR)
-}
+export const APM = new InjectionToken<ApmBase>('APM Base Client')
```

Comment:
Later versions of angular the builds were complaining about this being in the `.module.ts` file. In our package I pulled this injection token out to it's own file `apm.model.ts`. YMMV, but shouldn't be an issue for 9 though.

Reply:
Going to keep it as it is, will update if required.